src/Core/Framework/Resources/views/documents/includes/logo.html.twig

```diff
@@ -15,7 +15,7 @@ All blocks of this template are available in the template which renders this tem
         <header>
             {% block document_header %}
                 {% if config.logo %}
-                    <img src="{{ config.logo.url }}" class="logo"/>
+                    <img src="{{ config.logo.url }}" class="logo" alt="Logo"/>
```

Comment:
`Logo` is still meaningless. Just set it empty for decorative elements `alt=""`

Reply:
Setting the alt tag to null/empty can be a valid technique to mark images decorative as suggested by Malte. But it always looks kinda strange. ^^ 

https://www.w3.org/WAI/tutorials/images/decorative/